src/options.js

```diff
@@ -61,6 +61,7 @@ export type RealmOptions = {
   abstractValueImpliesMax?: number,
   arrayNestedOptimizedFunctionsEnabled?: boolean,
   reactFailOnUnsupportedSideEffects?: boolean,
+  rmModuleFuncs?: boolean,
```

Comment:
I don't like strange abbreviations. Let's call it `removeModuleFactoryFunctions`.

Reply:
wfm :-)